=== www/Zend/Config.php
==================================================================
--- www/Zend/Config.php	(revision 1088)
+++ www/Zend/Config.php	(local)
@@ -288,4 +288,10 @@
         $this->_extends[$extendingSection] = $extendedSection;
     }
 
+    public function __unset($name)
+    {
+        if ($this->_allowModifications) {
+            unset($this->_data[$name]);
+        }
+    }
 }
